setup.py

```diff
@@ -14,7 +14,9 @@
     author='Incuna',
     author_email='admin@incuna.com',
     url='https://github.com/incuna/incuna-test-utils/',
-    install_requires=[],
+    install_requires=[
+        'Pillow==2.8.1',
+    ],
```

Comment:
[For example](https://github.com/incuna/django-user-management/blob/852451a654f8e3c4f0651f67c4c90c694de03220/setup.py#L13-L20).